rllib/BUILD

```diff
@@ -338,6 +338,19 @@ py_test(
     args = ["--dir=tuned_examples/cql"]
 )
 
+py_test(
+    name = "learning_tests_pendulum_cql",
+    main = "tuned_examples/cql/pendulum_cql.py",
+    tags = ["team:rllib", "exclusive", "learning_tests", "torch_only", "learning_tests_cartpole", "learning_tests_discrete", "learning_tests_pytorch_use_all_core"],
+    size = "medium",
```

Comment:
size="large" is better, no?

Reply:
Yeah, maybe. It takes around 50 iterations.

Reply:
Will tune this in another PR.